README.md

```diff
@@ -27,46 +27,54 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>React Capstone Project</b></h3>
 
 </div>
 
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
+- [📖 About the Project](#about-project) 
+  - [🛠 Built With](#built-with) 
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
+  git clone <ssh link>
   - [Prerequisites](#prerequisites)
+   
   - [Install](#install)
+  npm install i 
+ 
+
   - [Usage](#usage)
+ run:  npm start
   - [Run tests](#run-tests)
+  run:  npm test
   - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
+
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
```

Comment:
- [ ] `required`
- [ ] I can't not see the recorded video presentation please add it as the requirements state.

Reply:
Video has been added